vscode/microsoft-kiota/CHANGELOG.md

```diff
@@ -0,0 +1,9 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+## [1.18.100000002] - 2024-09-12
+
+### Added
+
+- Extension Released for GA
```

Comment:
Let's not forget to update this of the new release version number

Reply:
Planning to do this update in a follow up PR shortly. There's just been alot of comments that have resolved it would be great to close this and reopen part 2. Especially because am not sure what our release version release is.